.github/labels.md

```diff
@@ -0,0 +1,249 @@
+may be taken.
+
+
+#### Label `category:bug`
+
+The Issue describes or solves a perceived malfunction within the game.
+
+#### Label `category:feature`
+
+The Pull Request contains a new game feature.  Features are parts of the game
+which a player can interact with e.g.: A new species, a new gameplay mechanic
+or a new command line switch for the server that changes gameplay behavior.
+
+#### Label `category:idea`
+
+??? No idea, overlaps with `category:feature` or `state:work in progress` IMO.
+
+#### Label `category:question`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:refactoring`
+
+The Pull Request contains a new or improved implementation of an existing
+feature or any other code restructuring that doesn't modify the current game
+mechanics or representation.
+
+#### Label `category:request`
+
+??? Requests don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+The Pull Request contains insignificant code changes that don't involve
+features, refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue deals with the python AI decision making code or affects it.
+
+#### Label `component:art music text assets`
+
+The Issue deals with creating game assets or verification if the assets match
+the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue deals with CMake, Visual Studio, Xcode or the general build process.
+
+#### Label `component:common`
+
+??? No idea.
+
+#### Label `component:deployment`
+
+The Issue deals with the deployment process or the software used create the
+installer.
+
+#### Label `component:documentation`
+
+The Issue deals with Doxygen or primarily contains documentation.
+
+#### Label `component:game mechanic`
+
+The Pull Request deals with the currently used or planned game mechanics.  The
+content of the Pull Request may require extensive gameplay balance testing.
+
+#### Label `component:infrastructure`
+
+The Pull Request deals with non-game related infrastructure that aids
+development.  For example this includes issues with the project homepage, used
+external services like continuous integration or hosting of release binaries.
+
+#### Label `component:internal`
+
+??? No idea.  Thread suggest it's a non-user visible feature, but then it's the
+wrong category.
+
+#### Label `component:network`
+
+The Issue deals with TCP/IP networking and establishing a test network
+infrastructure.
+
+#### Label `component:scripting`
+
+The Pull Request deals with the FOCS language and probably game mechanics.
+Probably most used in conjunction with `component:game mechanic` and
+`component:art music text assets`.
+
+#### Label `component:translation`
+
+The Pull Request deals with the game style, game terminology, stringtable syntax
+and the translation process in general.  It maybe helpful to know the language
+of the translation to proofread.
```

Comment:
Just to clarify: so this label not only covers the actual translations (non English stringtables), but also all changes/additions to the English stringtable as well? So if a PR consists of a change to some in-game text, not a translation per se, this labels applies too? Have never been sure about that...